### DISCUSSION TITLE
S1 Orbits

### DISCUSSION THREAD

**Discussion by alevasca** [isce-framework/isce2#572] [Q&A]
Hello everyone,
I'm working on a stack with the script **stackSentinel.py**. So far I know, we can use **fetchOrbit.py** to get the orbit files. However, it doesn't work as the connection with https://qc.sentinel1.eo.esa.int/aux_resorb/ fails. Anyone could tell me how to download these files? Thanks!
--------------------

**orbits$>** fetchOrbit.py -i ../slc/S1A_IW_SLC__1SDV_20211212T050500_20211212T050527_040971_04DDD1_8E66.zip -o ./
Reference time:  2021-12-12 05:05:27
Satellite name:  S1A
https://qc.sentinel1.eo.esa.int/aux_poeorb/?validity_start=2021-12-10..2021-12-14&sentinel1__mission=S1A
Querying for precise orbits
Exception - something went wrong with the web scraper:
Exception: HTTPSConnectionPool(host='qc.sentinel1.eo.esa.int', port=443): Max retries exceeded with url: /aux_poeorb/?validity_start=2021-12-10..2021-12-14&sentinel1__mission=S1A (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f38d72a6950>: Failed to establish a new connection: [Errno 11] Resource temporarily unavailable'))
Continuing process
Failed to find precise orbits for Time 2021-12-12 05:05:27
https://qc.sentinel1.eo.esa.int/aux_resorb/?validity_start=2021-12-11..2021-12-13&sentinel1__mission=S1A
Querying for restituted orbits
Exception - something went wrong with the web scraper:
Exception: HTTPSConnectionPool(host='qc.sentinel1.eo.esa.int', port=443): Max retries exceeded with url: /aux_resorb/?validity_start=2021-12-11..2021-12-13&sentinel1__mission=S1A (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f38d72a67d0>: Failed to establish a new connection: [Errno 11] Resource temporarily unavailable'))
Continuing process
Failed to find restituted orbits for Time 2021-12-12 05:05:27

**Comment by scottstanie** [comment]
Do you have an old version of isce installed? I believe the URL was changed from that old one about a year ago (https://github.com/isce-framework/isce2/commit/2f8de43d280687c8bb0f349b8b3a96fcad51edf6). 
The current fetchOrbit.py points to the correct URL, so you could try re-installing/updating isce

https://github.com/isce-framework/isce2/blob/main/contrib/stack/topsStack/fetchOrbit.py#L11

**Reply by alevasca** [reply]
Yes, indeed! I will install the last one. Thank you!

**Comment by riyasmjgeo** [comment]
If you couldn't fix it, there is an out-of-box solution https://github.com/scottstanie/sentineleof

**Reply by alevasca** [reply]
Thank you! At first, I will install the last version and see what happens. If it doesn't work I will try this solution.
